karate-core/src/main/java/com/intuit/karate/core/Config.java

```diff
@@ -285,6 +292,45 @@ public boolean configure(String key, Variable value) { // TODO use enum
                 return true;
             case "localAddress":
                 localAddress = value.getAsString();
+                return true;
+            case "continueOnStepFailure":
+                continueOnStepFailureMethods.clear(); // clears previous configuration - in case someone is trying to chain these and forgets resetting the previous one
+
+                boolean enableContinueOnStepFailureFeature = false;
+                Boolean continueAfterIgnoredFailure = null;
+
+                Collection<Method> selectedMethods = new HashSet<>();
+                if (value.isMap()) {
+                    Map<String, Object> map = value.getValue();
+                    List<String> stepKeywords = (List<String>) map.get("keywords");
+                    if (stepKeywords != null) {
+                        selectedMethods = StepRuntime.findMethodsByKeywords(stepKeywords);
+                    }
+
+                    continueAfterIgnoredFailure = (Boolean) map.get("continueAfter");
+                    enableContinueOnStepFailureFeature = map.get("enabled") != null && (Boolean) map.get("enabled");
+                }
+
+                if (value.isTrue() || enableContinueOnStepFailureFeature) {
+
+                    if(selectedMethods.isEmpty()) {
+                        selectedMethods = StepRuntime.findMethodsByKeyword("match");
```

Comment:
can be potentially a static variable for performance

Reply:
just a general comment - one reason to try avoid reflection is because in the future it makes it easier to convert karate into a native binary using graal which will be mind-blowing in a lot of ways

that said we already use reflection in a few places. but just wanted to document this "principle" I had in mind

Reply:
That's a good principle, I can see the advantages. Will keep it in this PR but good principle to keep in mind for future.

Reply:
> can be potentially a static variable for performance

not sure I entirely follow you but simplified the code and removed the need for the "intermediary" collection variable selectedMethods

Reply:
sorry I mean that `StepRuntime.findMethodsByKeyword("match")` can be done in a static initializer and stored in a static variable `StepRuntime.METHOD_MATCH` etc